server/src/main/java/io/crate/metadata/IndexReference.java

```diff
@@ -99,7 +105,7 @@ public IndexReference(int position,
                           IndexType indexType,
                           List<Reference> columns,
                           @Nullable String analyzer) {
-        super(ident, RowGranularity.DOC, DataTypes.STRING, ColumnPolicy.DYNAMIC, indexType, false, true, position, null);
+        super(ident, RowGranularity.DOC, DataTypes.STRING, ColumnPolicy.DYNAMIC, indexType, false, false, position, null);
         this.columns = columns;
```

Comment:
Revealed by the new `IndexReferenceTest`:

FT column is always false on creating metadata:
```
if (definition.docValues() != definition.dataType.storageSupport().getComputedDocValuesDefault(definition.indexType)) {
            // definition.docValues falls back to default if not specified.
            // If computed value is non-default it means doc values are supported but disabled.
            mapping.put(DOC_VALUES, "false");
        }
```

and StorageSupport explicitly returns false for FULLTEXT indices (visitIndexDefinition -> setAnalyzer sets FULTEXT on TABLE LEVEL index definition).

Or I can use another ctor and pass value we got from the metadata (it will be false anyway, though)